engine/src/main/java/org/terasology/rendering/world/WorldRendererLwjgl.java

```diff
@@ -179,24 +178,12 @@ public boolean pregenerateChunks() {
     @Override
     public void update(float deltaInSeconds) {
 
-        updateTick(deltaInSeconds);
-
-        playerCamera.update(deltaInSeconds);
-        positionShadowMapCamera();
-        shadowMapCamera.update(deltaInSeconds);
-
-        renderableWorld.update();
```

Comment:
This is stuff that I *think* needs to be updated only once per frame rather than once per tick. It is now in preRenderUpdate().